utilities/utilities.py

```diff
@@ -134,7 +132,6 @@ def __init__(self):
             intents.members = True
             intents.message_content = True
             self.client = discord.Client(intents=intents)
-            self.wiki = SemanticWiki(wiki_config["uri"], wiki_config["user"], wiki_config["password"])
```

Comment:
this is the line I needed to remove to be able to move `stampy.ai` to the UI instead of the wiki ... everything else is a tangled web of dependencies to this initial login to wiki